### PR TITLE
Kernel copy func

### PR DESCRIPTION
**Pull request by Alexander-Makaryev**

### CODE REVIEW

File: dpnp/backend/kernels/dpnp_krnl_elemwise.cpp [hunk 1]
```diff
@@ -285,32 +285,38 @@ void dpnp_arange_c(size_t start, size_t step, void* result1, size_t size)
...
+    fmap[DPNPFuncName::DPNP_FN_ARANGE][eft_DBL][eft_DBL] = {eft_DBL, (void*)dpnp_arange_c<double>};
 
-    fmap[DPNPFuncName::DPNP_FN_CONJIGUATE][eft_C128][eft_C128] = {eft_C128,
-                                                                  (void*)dpnp_conjugate_c<std::complex<double>>};
-    fmap[DPNPFuncName::DPNP_FN_CONJIGUATE][eft_DBL][eft_DBL] = {eft_DBL, (void*)dpnp_copy_c<double>};
-    fmap[DPNPFuncName::DPNP_FN_CONJIGUATE][eft_FLT][eft_FLT] = {eft_FLT, (void*)dpnp_copy_c<float>};
     fmap[DPNPFuncName::DPNP_FN_CONJIGUATE][eft_INT][eft_INT] = {eft_INT, (void*)dpnp_copy_c<int>};
     fmap[DPNPFuncName::DPNP_FN_CONJIGUATE][eft_LNG][eft_LNG] = {eft_LNG, (void*)dpnp_copy_c<long>};
+    fmap[DPNPFuncName::DPNP_FN_CONJIGUATE][eft_FLT][eft_FLT] = {eft_FLT, (void*)dpnp_copy_c<float>};
+    fmap[DPNPFuncName::DPNP_FN_CONJIGUATE][eft_DBL][eft_DBL] = {eft_DBL, (void*)dpnp_copy_c<double>};
```
Comment (densmirn) [inline dpnp/backend/kernels/dpnp_krnl_elemwise.cpp:296]: We need to replace `dpnp_copy_c` with `dpnp_conjugate_c` here.

Comment (Alexander-Makaryev) [reply dpnp/backend/kernels/dpnp_krnl_elemwise.cpp:296]: No, we don't need to.)

**Comment by shssf** [comment]
need tests passed

**Comment by Alexander-Makaryev** [comment]
> need tests passed

External tests are failed on master branch too. Looks like it is not related to this PR.

**Comment by Alexander-Makaryev** [comment]
> need tests passed

@shssf Tests are passed

**Review by densmirn — APPROVED** [review]

**Review by PokhodenkoSA — APPROVED** [review]
LGTM

**Pull request merged**